tests/src/system/basic/WskBasicTests.scala

```diff
@@ -487,16 +487,16 @@ class WskBasicTests
             withActivation(wsk.activation, run) {
                 activation =>
                     activation.response.result shouldBe Some(dynamicParams.toJson)
-                    activation.duration shouldBe 0L // shouldn't exist but CLI generates it
-                    activation.end shouldBe Instant.EPOCH.toEpochMilli // shouldn't exist but CLI generates it
+                    activation.duration shouldBe 0L         // shouldn't exist but CLI generates it
+                    activation.end shouldBe Instant.EPOCH   // shouldn't exist but CLI generates it
```

Comment:
did i ask this already? the comment says "shouldn't exist" but CLI generates it - is this not being removed/addressed by this pr?